crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -276,81 +277,93 @@ pub(crate) fn create(
                 python_home,
             )?;
 
-            if interpreter.markers().implementation_name() == "graalpy" {
-                copy_launcher_windows(
-                    WindowsExecutable::GraalPy,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PythonMajor,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-            } else {
-                copy_launcher_windows(
-                    WindowsExecutable::Pythonw,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-            }
-
-            if interpreter.markers().implementation_name() == "pypy" {
-                copy_launcher_windows(
-                    WindowsExecutable::PythonMajor,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PythonMajorMinor,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PyPy,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PyPyMajor,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PyPyMajorMinor,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PyPyw,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PyPyMajorMinorw,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
+            match interpreter.markers().implementation_name() {
```

Comment:
Nit: This is available without `markers()`.

I'd also consider using `LenientImplementationName`

Reply:
Fixed. I left it as-is for now w/r/t `LenientImplementationName`, I'd just rather not change it unless we have a reason to.